specs/commandline/sign.md

```diff
@@ -155,6 +157,20 @@ Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(
 Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
 
+### Sign an OCI artifact and timestamp the signature with user specified timestamp authority
+
+```shell
+# Prerequisites:
+# A default signing key is configured using CLI "notation key".
+# Signer knows the TSA url that they want to use to require a RFC 3161 timestamp.
+# Signer has downloaded the TSA's root certificate in their file system.
+
+# Use option "--tsa-url" to specify the timestamp authority URL.
+# Use option "--tsa-root-cert" to specify the filepath of the tsa root
+# certificate.
+notation sign --tsa-url <tsa_url> --tsa-root-cert <tsa_root_certificate_filepath> <registry>/<repository>@<digest>
```

Comment:
Does the flag `--tsa-root-cert` allow an URL when passing a root cert? 

For example, users are running notation in a CI/CD platform and its runner machine is a temporary VM resource, users have to download the root cert into the runner machine filesystem every time before every signing action. It might not be efficient and performant in this case.

Reply:
Then that's another network roundtrip for Notation.

For a CI/CD pipeline signing, user does NOT have to download the root cert every time. They could simply download it once and save it in the repo, the CI/CD pipeline could retrieve the cert from the repo. For example, in GitHub workflow, this is done by [`actions/checkout`](https://github.com/actions/checkout).